<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkExchangeWindow</class>
 <widget class="QWidget" name="NetworkExchangeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="exchangePortLabel">
     <property name="text">
      <string>Exchange &amp;Port:</string>
     </property>
     <property name="buddy">
      <cstring>exchangePortSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="exchangePortSpinBox">
     <property name="specialValueText">
      <string>Random</string>
     </property>
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="correctionMode">
      <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="deviceNameLabel">
     <property name="text">
      <string>Device &amp;Name:</string>
     </property>
     <property name="buddy">
      <cstring>deviceNameLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="deviceNameLineEdit">
     <property name="placeholderText">
      <string>Name shown to other devices</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="exchangeCheckBox">
     <property name="text">
      <string>&amp;Exchange active:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QTreeView" name="treeView">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="animated">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exchangeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>treeView</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
